Automattic/woocommerce-payments
# Added a notice about custom checkout fields conflicting with express checkout

Fixes #8299

#### Changes proposed in this Pull Request

Custom fields are currently incompatible with express checkout methods. This PR adds a notice next to the Apple Pay and Google Pay settings when a change in the checkout fields is detected.

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/02747aa2-32cc-43d8-9a8f-4eac18b592da)

![image](https://github.com/Automattic/woocommerce-payments/assets/12385501/c68a7486-1b10-4231-85f8-742a916988d4)


#### Testing instructions

1. Go to WooPayments settings and notice there is no warning next to Google Pay / Apple Pay toggle section.
2. Go to Google Pay / Apple Pay page settings and notice there is no warning.
3. Install an extension to change/add checkout fields, or do it manually by using the filters `woocommerce_billing_fields`, `woocommerce_shipping_fields`, and `woocommerce_checkout_fields`.
4. Go back to 1. and 2. and ensure there is a warning.

-------------------


**Post merge**
